deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -34,11 +34,11 @@ spec:
             - "--v=5"
           env:
             - name: ADDRESS
```

Comment:
For *all* ADDRESS environment variables and its counterpart volume mounts named "socket-dir" we should drop the explicit naming scheme using the drivername/pluginname. These are local to the container sockets and directories for use, and do not need a naming scheme that includes the drivername.

Comment applies to all files under deploy.

I say this to reduce amount of changes required to such deployment YAMLs when there is a need to deploy multiple instances of Ceph-CSI drivers, with different leading driver names. (e.g rook-instance-1-cephfsplugin.csi.ceph.com and rook-instance-1-cephfsplugin.csi.ceph.com, for CSI naming when dealing with multiple CSI instances see https://github.com/rook/rook/pull/2677/files#r261420176 )

Reply:
@ShyamsundarR can you open a separate issue for this one?

Reply:
I thought the change can be done along with this one, as it is related and touches exactly the same set of lines that a patch addressing this would.

Reply:
IMO these also require discussion, what was the idea behind doing the things in the way as there are now, and also what advantage we are going to get by changing things, last but not least requires some amount of testing and validation. I would like to take it up as a separate task

Reply:
Fine, we can deal with this in a separate PR.